nautobot/core/templates/inc/footer.html

```diff
@@ -27,7 +27,7 @@
                         </p>
                     {% endif %}
                 </div>
-                <div class="col-xl-4 col-lg-4 col-md-4 col-sm-3 col-xs-12 text-center footer-col-2">
+                <div class="col-xl-4 col-lg-4 col-md-3 col-sm-3 col-xs-12 text-center footer-col-2">
```

Comment:
The three segments of the footer should always sum to 12 columns, so if you're decreasing this one you should increase another one accordingly. I think they all need to be adjusted a bit to avoid overflow. Try this set of classes and see what you think as the window size changes:

```
class="col-xl-4 col-lg-4 col-md-3 col-sm-6 col-xs-12 text-left footer-col-1 no-wrap"
class="col-xl-4 col-lg-3 col-md-3 col-sm-6 col-xs-12 text-center footer-col-2"
class="col-xl-4 col-lg-5 col-md-6 col-sm-12 col-xs-12 text-right footer-col-3 noprint"
```

(If making this change, probably also need some changes to the media queries above so that columns are aligned appropriately at the "sm" breakpoint - currently here they look like this:)

![image](https://github.com/nautobot/nautobot/assets/5603551/c5671d83-e12a-4876-a0bc-c7300d3ebbdb)

Reply:
Ok.  Will fine tune it.  I agree this is the desired state with bootstrap rows/columns.  That said, when I have it as interval of 12 the 3rd item always seems to overflow behind the row length.  I believe it is due to a nowrap that impacts the medium width views.  I don’t want to fix it by removing that however because then the menu buttons on the bottom right begin to wrap at larger width scenarios which is undesirable.  

I’ll get the magic combination for medium width (md) column nailed down as part of this PR.

Reply:
Did my proposed changes for the `lg`, `md`, and `sm` column widths above end up not working out or being unnecessary?

Reply:
I’m pretty sure it helped with right hand buttons overflowing, but had a negative impact to the other two columns crashing into each other.  I’ll try again and validate.